tools/internal_ci/linux/grpc_bazel_rbe.sh

```diff
@@ -34,3 +34,7 @@ bazel_rbe/bazel_wrapper \
   $BAZEL_FLAGS \
   "$@" \
   -- ${BAZEL_TESTS:-//test/...}
+
+# Print disk usage
+df -h
+du -h --max-depth=1 * | sort -rh | head -100
```

Comment:
should be `du -h --max-depth=1 .` ... as it is now, this will report 2 folders deep, and miss hidden folders in the cwd (I believe).

Reply:
you're right